quickfort.lua

```diff
@@ -35,19 +34,19 @@ Usage:
 **quickfort gui**
     Starts the quickfort dialog, where you can run blueprints from an
     interactive list.
-**quickfort list [-m|--mode <mode>] [-l|--library] [-h|--hidden] [search string]**
+**quickfort list [-m|-\-mode <mode>] [-l|-\-library] [-h|-\-hidden] [search string]**
```

Comment:
Good catch! I would ordinarily suggest preformatted text here (double backticks instead of asterisks), but it looks like that would wrap, and the console help text is coming from a different source (i.e. users wouldn't see the `-\-` when running `quickfort help` and try to copy it).

![image](https://user-images.githubusercontent.com/3719547/106371893-08175f80-6338-11eb-8d7b-aa8eb80a4d05.png)